kinto/scripts/command.py

```diff
@@ -0,0 +1,145 @@
+import ConfigParser
+import sys
+import os,binascii
+
+cfgfile = open("kinto/config/kinto.ini",'w') #enter path of file
```

Comment:
This mean that you open the file with `write` mode. We probably don't want to erase it since it is under git control. Every time we will pull the changes with git, it will be overwritten.

I suggest that we read it but write to another one (like `config/dev.ini`)